app/scripts/controllers/membership.js

```diff
@@ -88,16 +91,16 @@ angular
           .then(function() {
             resetForm();
             refreshRoleBindingList();
-            showAlert('rolebindingCreate', 'success', messages.update.subject.success({
+            showAlert('rolebindingCreate', 'success', escaped.update.subject.success({
```

Comment:
> "Bob" was given the role "view". "Bob" needs to be escaped, correct?

No, Angular will escape any special characters you have in the string since the alerts use angular expressions.

https://github.com/openshift/origin-web-console/blob/master/app/views/_alerts.html#L22

We only need to escape values that we pass to `ng-bind-html` in the dialog.